.github/workflows/test-jpf.yml

```diff
@@ -0,0 +1,41 @@
+name: Run TLC JPF Tests
+# These test take a long time to run. We only want to run them whenever the tested classes are updated.
+
+on:
+  push:
+    paths:
+      - 'tlatools/org.lamport.tlatools/src/tlc2/tool/fp/OffHeapDiskFPSet.java'
+      - 'tlatools/org.lamport.tlatools/test-verify/tlc2/tool/fp/OffHeapDiskFPSetJPFTest.java'
+      - 'tlatools/org.lamport.tlatools/src/tlc2/tool/queue/StateQueue.java'
+      - 'tlatools/org.lamport.tlatools/test-verify/tlc2/tool/queue/StateQueueJPFTest.java'
+  pull_request:
+    paths:
+      - 'tlatools/org.lamport.tlatools/src/tlc2/tool/fp/OffHeapDiskFPSet.java'
+      - 'tlatools/org.lamport.tlatools/test-verify/tlc2/tool/fp/OffHeapDiskFPSetJPFTest.java'
+      - 'tlatools/org.lamport.tlatools/src/tlc2/tool/queue/StateQueue.java'
+      - 'tlatools/org.lamport.tlatools/test-verify/tlc2/tool/queue/StateQueueJPFTest.java'
+
+jobs:
+  run-tlc-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: '11'
```

Comment:
Do we need to upgrade jpf if this runs on Java11?

Reply:
Yes, in this pr I'm also adding the latest version of JPF jar